### PR TITLE
Refactor ConsensusCommitUtils

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java [hunk 1]
```diff
@@ -183,26 +170,20 @@ public static boolean isTransactionalMetaColumn(String columnName, Set<String> a
...
-   *
-   * @param columnName a column name
-   * @param allColumnNames a set of all the column names
-   * @return whether the specified column is transactional or not
-   */
-  public static boolean isBeforeImageColumn(String columnName, Set<String> allColumnNames) {
-    if (!allColumnNames.contains(columnName)) {
+    if (!tableMetadata.getColumnNames().contains(columnName)
+        || tableMetadata.getPartitionKeyNames().contains(columnName)
+        || tableMetadata.getClusteringKeyNames().contains(columnName)) {
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java:175]: I also added a condition where primary key columns are not before image columns.

File: core/src/test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtilsTest.java [hunk 2]
```diff
@@ -371,53 +360,101 @@ public void isBeforeImageColumn_ResultGiven_shouldReturnProperResult() {
...
+    assertThat(ConsensusCommitUtils.isBeforeImageColumn(Attribute.BEFORE_COMMITTED_AT, metadata))
         .isTrue();
-    assertThat(
-            ConsensusCommitUtils.isBeforeImageColumn(Attribute.BEFORE_PREPARED_AT, allColumnNames))
+    assertThat(ConsensusCommitUtils.isBeforeImageColumn("aaa", metadata)).isFalse();
+  }
+
+  @Test
+  public void
+      isBeforeImageColumn_PartitionKeyAndClusteringKeyWithBeforePrefixGiven_shouldReturnProperResult() {
```
Comment (brfrn169) [inline core/src/test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtilsTest.java:409]: Added a test for the new  condition.

File: core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java [hunk 3]
```diff
@@ -158,21 +158,8 @@ public static TableMetadata removeTransactionalMetaColumns(TableMetadata tableMe
    * @return whether the specified column is a transactional meta column or not
    */
   public static boolean isTransactionalMetaColumn(String columnName, TableMetadata tableMetadata) {
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java:160]: I refactored the logic of `isTransactionalMetaColumn()` and `isBeforeImageColumn()` and `isAfterImageColumn()`.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**